lib/correios/frete/pacote.rb

```diff
@@ -48,8 +48,11 @@ def calcular_medidas(item)
           @largura = item.largura
           @altura = item.altura
         else
-          dimensao = @volume.to_f**(1.0/3)
-          @comprimento = @largura = @altura = dimensao
+          dimensoes = @itens.map { |item| [item.comprimento, item.largura, item.altura] }.flatten
+          dimensao_maxima = dimensoes.max
+          dimensao = (@volume.to_f/dimensao_maxima)**0.5
+          @comprimento = dimensao_maxima
+          @largura = @altura = dimensao
```

Comment:
What this change is about? Not related with the new service codes, right?

Please open a new pull request for that and explain your intention.